packages/server/graphql/types/ReflectPrompt.ts

```diff
@@ -1,22 +1,22 @@
 import {GraphQLBoolean, GraphQLID, GraphQLInterfaceType, GraphQLNonNull} from 'graphql'
 import {RETRO_PHASE_ITEM} from 'parabol-client/utils/constants'
 import RetroPhaseItem from './RetroPhaseItem'
-import CustomPhaseItemTypeEnum from './CustomPhaseItemTypeEnum'
+import ReflectPromptTypeEnum from './ReflectPromptTypeEnum'
 import Team from './Team'
 import {resolveTeam} from '../resolvers'
 import GraphQLISO8601Type from './GraphQLISO8601Type'
 
-export const customPhaseItemFields = () => ({
+export const reflectPromptFields = () => ({
   id: {
     type: new GraphQLNonNull(GraphQLID),
     description: 'shortid'
   },
   createdAt: {
     type: new GraphQLNonNull(GraphQLISO8601Type)
   },
-  phaseItemType: {
```

Comment:
let's leave this here & add a `deprecationReason` to it. that way folks who haven't upgraded to the latest version won't get an error (and possibly a crash)

Reply:
Should I restore the CustomPhaseItem.ts and CustomPhaseItemTypeEnum.ts files? Or just deprecate these variables in the new ReflectPrompt files?